lua/wikis/splatoon/MatchSummary.lua

```diff
@@ -5,120 +5,108 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
-local CustomMatchSummary = {}
-
 local Lua = require('Module:Lua')
 
 local Array = Lua.import('Module:Array')
-local MapTypeIcon = Lua.import('Module:MapType')
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
 local Operator = Lua.import('Module:Operator')
-local String = Lua.import('Module:StringUtils')
-local Table = Lua.import('Module:Table')
 local WeaponIcon = Lua.import('Module:WeaponIcon')
 
-local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
-local MatchSummary = Lua.import('Module:MatchSummary/Base')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
 local MatchSummaryWidgets = Lua.import('Module:Widget/Match/Summary/All')
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
 local WidgetUtil = Lua.import('Module:Widget/Util')
 
-local NON_BREAKING_SPACE = '&nbsp;'
+
+local CustomMatchSummary = {}
 
 ---@param args table
 ---@return Html
 function CustomMatchSummary.getByMatchId(args)
-	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {width = '490px', teamStyle = 'bracket'})
+	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {width = '500px', teamStyle = 'bracket'})
+end
+
+---@param match MatchGroupUtilMatch
+---@return Widget[]
+function CustomMatchSummary.createBody(match)
+	return WidgetUtil.collect(
+		Array.map(match.games, function(game, gameIndex)
+			return CustomMatchSummary._createGameRow(game, gameIndex, match.game)
+		end)
+	)
 end
 
----@param date string
 ---@param game MatchGroupUtilGame
 ---@param gameIndex integer
+---@param gameTitle string
 ---@return Widget?
-function CustomMatchSummary.createGame(date, game, gameIndex)
-	local weaponsData = Array.map(game.opponents, function(opponent)
-		return Array.map(opponent.players, Operator.property('weapon'))
-	end)
+function CustomMatchSummary._createGameRow(game, gameIndex, gameTitle)
```

Comment:
annos not matching the function
if we can use game.game just revert this one line to its old version and use game.game in line 62 (instead of gameTitle)